server/src/authorization/index.ts

```diff
@@ -80,6 +84,8 @@ function isAllowedByEventRole(
 }
 
 function getUserPermissionsForInstance(user: User): string[] {
+  console.log('checking instance permissions');
+  console.log('user', user);
```

Comment:
may be better to remove this console.log, if it not necessary

Reply:
Thanks, they were just debugging tools, so I should have removed them.